bndtools.core/src/bndtools/editor/common/Buttons.java

```diff
@@ -0,0 +1,81 @@
+package bndtools.editor.common;
+
+import org.bndtools.core.ui.icons.Icons;
+import org.eclipse.jface.action.Action;
+import org.eclipse.jface.action.ActionContributionItem;
+import org.eclipse.jface.action.IAction;
+import org.eclipse.swt.program.Program;
+
+/**
+ * Helper containing help buttons for different parts / views which link to the
+ * user manual.
+ */
+public final class Buttons {
```

Comment:
Isn't this HelpButtons? You could call it Buttons but then imho it should contain all buttons and the functions should not be called *Help*

BTW, I like how you separate the data from the the code. However, if you make a function like createHelpButton, next time consider moving it in a utility in bndtools.utils package. Note that it is not related to Help, it is just a useful function to make an action. This makes this class smaller and more readable and might be used in other places.

* You could make this an interface so you do not have to clutter the class with 'public static final', that is implied in an interface.
* We're using Java 17 so you can use the """ comments to make the text easier to read

I am ok to keep it here, these changes are not so important but for future PRs.

Reply:
Thanks @pkriens for the suggestions. During dev I always wonder how to call what and where to put what :)
I did the renaming to HelpButtons and moved the methods to EditorUtils (feel free to put it somewhere else if you like).

For the """ Textblocks: I tried but decided against it because the code formatter always changed the indentation which aftected display of the tooltips negatively. Maybe this is an item for another PR.

Reply:
About where to put what ... I generally have the rule of cohesion. Each aggregate (method,class,package,bundle) should have a clear focus. A class with a util (like the action methods) and the data is not cohesive. And whenever you can make a function reusable without effort, do it. It might never be reused but it will at least be easier to understand without the context of parts that are not relevant for that function.

Anyway, thanks for all the work!